frog/imports/ui/Body.jsx

```diff
@@ -1,15 +1,16 @@
-import React, { Component, PropTypes } from 'react';
+<<<<<<< HEAD
+﻿import React, { Component, PropTypes } from 'react';
 
 import Repository from './repository/Repository.jsx';
-import Editor from './editor/Editor.jsx';
-import Engine from './Engine.jsx';
+import Editor from './Editor.jsx';
```

Comment:
The component `Editor` is in `./editor/Editor.jsx`. I think you wrongly resolved the conflicts